airflow/providers/amazon/aws/hooks/ses.py

```diff
@@ -0,0 +1,79 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+This module contains AWS SES Hook
+"""
+from typing import Iterable, List, Optional, Union
+
+from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+from airflow.utils.email import build_mime_message
+
+
+class SESHook(AwsBaseHook):
+    """
+    Interact with Amazon Simple Email Service.
+
+    Additional arguments (such as ``aws_conn_id``) may be specified and
+    are passed down to the underlying AwsBaseHook.
+
+    .. seealso::
+        :class:`~airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`
+    """
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **{**kwargs, **{'client_type': 'ses'}})
```

Comment:
I did this to fix an error reported by `mypy` pre-commit hook that was reporting the error below:
```
removed directory './airflow/__pycache__'
airflow/providers/amazon/aws/hooks/ses.py:39: error: "__init__" of
"AwsBaseHook" gets multiple values for keyword argument "client_type"
            super().__init__(client_type='ses', *args, **kwargs)
            ^
Found 1 error in 1 file (checked 1 source file)
###########################################################################################
                   EXITING /opt/airflow/scripts/ci/in_container/run_mypy.sh WITH STATUS CODE 1
###########################################################################################
```